G33 minor change

LVD-AC opened the pull request:
- further named remaining tests
- A and O  parameters obsolete
- quick homing with home_delta()

LVD-AC committed:
Merge remote-tracking branch 'refs/remotes/MarlinFirmware/RCBugFix' into RCBugFix

LVD-AC committed:
Merge remote-tracking branch 'refs/remotes/MarlinFirmware/RCBugFix' into RCBugFix

LVD-AC committed:
Minor change to G33

- use home_delta() instead of home_all_axis
- further naming tests
- getting rid of O and A parameters: always quick P1
- one T parameter: changed to bool -> T1 (default when ommited) tower
angles on, T0 tower angles off

LVD-AC commented on Marlin/Marlin_main.cpp:
```diff
@@ -5103,16 +5099,20 @@ void home_all_axes() { gcode_G28(); }
-
-      home_all_axes();
+      #if HOTENDS > 1
+        const uint8_t old_tool_index = active_extruder;
```
this line is obsolete

thinkyhead replied:
You have my leave to remove it.

thinkyhead commented on Marlin/Marlin_main.cpp:
```diff
@@ -5103,16 +5099,20 @@ void home_all_axes() { gcode_G28(); }
+      stepper.synchronize();
       #if HAS_LEVELING
-        set_bed_leveling_enabled(false);
+        reset_bed_level; //after calibration bed-level will no longer be valid
```
What about `reset_bed_level`? To call it, add parentheses `()`.

LVD-AC replied:
ok, i'll do so tomorrow

LVD-AC committed:
thinkyhead remarks

thinkyhead commented on Marlin/Marlin_main.cpp:
```diff
@@ -5412,11 +5411,14 @@ void home_all_axes() { gcode_G28(); }
         stepper.synchronize();
-
-        home_all_axes();
+        endstops.enable(true);
```
`setup_for_endstop_or_probe_move` perhaps here? And after homing, `clean_up_after_endstop_or_probe_move`… possibly… See if these functions do the right thing for this situation.

thinkyhead replied:
In general, after enabling endstops to do something, when done the method to call is `endstops.not_homing()` which will set them back to their global non-homing state.

thinkyhead committed:
Readme Version 1.1.0

thinkyhead committed:
bugfix-1.1.x ReadMe, Version, helper scripts

LVD-AC committed:
Thinkyhead comments

bgort committed:
fix error in M105 output -- use SERIAL_PROTOCOL for ints instead of SERIAL_PROTOCOL_F (#6584)

also removed wayward 'address of' ampersand in setTargetHotend and setTargetBed parameters

Commit:
Fixed USE_CONTROLLER_FAN #define usage

LVD-AC committed:
correction

thinkyhead committed:
Minor fix in K8200 readme

thinkyhead committed:
Tweak git helper scripts

thinkyhead committed:
Merge pull request #6587 from zlowred/bugfix-1.1.x

Fixed USE_CONTROLLER_FAN #define usage

LVD-AC committed:
Merge remote-tracking branch 'refs/remotes/MarlinFirmware/bugfix-1.1.x' into LVD_delta_autocal

LVD-AC committed:
Merge remote-tracking branch 'refs/remotes/MarlinFirmware/bugfix-1.1.x' into LVD_delta_autocal

thinkyhead committed:
Tweak command index increment

LVD-AC committed:
credit

LVD-AC committed:
Thinkyhead comments amendment

LVD-AC committed:
Preventing premature end

thinkyhead committed:
Edit configuration comments

thinkyhead committed:
Merge pull request #6600 from thinkyhead/bugfix_first_fixes

Update git helper scripts, config comments

thinkyhead committed:
Fix mfrb and mfpub

LVD-AC committed:
Merge remote-tracking branch 'refs/remotes/MarlinFirmware/bugfix-1.1.x' into LVD_delta_autocal

LVD-AC commented:
Squashed and cleaned in #6621
